src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/xunitrunner-perf/csvjsonconvertor.py

```diff
@@ -0,0 +1,200 @@
+    manufacturer.manufacturerName = opts['--manufacturerName']
+    machinepool.manufacturer = manufacturer
+
+    microarch = serialobj.MicroArch()
+    microarch.microarchName = opts['--microarchName']
+
+    osInfo = serialobj.OSInfo()
+    osInfo.osInfoName = opts['--osInfoName']
+    osInfo.osVersion = opts['--osVersion']
+
+    machinepool.osInfo = osInfo
+    machinepool.microarch = microarch
+    machinepool.NumberOfCores = opts['--numberOfCores']
+    machinepool.NumberOfLogicalProcessors = opts['--numberOfLogicalProcessors']
+    machinepool.TotalPhysicalMemory = opts['--totalPhysicalMemory']
+    machinepool.machinepoolName = opts['--machinepoolName']
+    machinepool.machinepoolDescription = opts['--machinepoolDescription']
+    machinepool.TotalPhysicalMemory = opts['--totalPhysicalMemory']
+    run.machinepool = machinepool
+
+    config = serialobj.Config()
+    config.configName = opts['--configName']
+    run.config = config
+
+
+    runs = list()
+    runs.append(run)
+
+    job = serialobj.Job()
+    job.Runs = runs
+
+    user = serialobj.User()
+    user.userName = opts['--username']
+    user.userAlias = opts['--userAlias']
+    job.user = user
+
+    buildInfo = serialobj.BuildInfo()
+    buildInfo.buildInfoName = opts['--buildInfoName']
+    buildInfo.buildNumber = opts['--buildNumber']
+    buildInfo.branch = opts['--branch']
+    job.buildInfo = buildInfo
+
+    jobType = serialobj.JobType()
+    jobType.jobTypeName = opts['--jobTypeName']
+    job.jobType = jobType
+
+    jobGroup = serialobj.JobGroup()
+    jobGroup.jobGroupName = opts['--jobGroupName']
+    job.jobGroup = jobGroup
+
+    job.jobDescription = opts['--jobDescription']
+    job.jobName = opts['--jobName']
+
+    root = serialobj.Root()
+    root.job = job
+    jsonOutput = serialobj.JsonOutput()
+    jsonOutput.roots.append(root)
+    with open(opts['--jsonFile'], 'w+') as opfile:
+        opfile.write(jsonOutput.to_JSON())
+        opfile.flush()
+        opfile.close()
+
+    log.info('Conversion of csv to json successful')
+
+def check_required_args(optdict):
+    if not '--csvFile' in optdict or not '--jobName' in optdict or not '--configName' in optdict or not '--userAlias' in optdict or not '--branch' in optdict or not '--osInfoName' in optdict or not '--jsonFile' in optdict:
+        raise getopt.GetoptError
+
+def validate_args(args):
+    # validate required params
+    opts, args = getopt.getopt(args, '', ['csvFile=', 'jsonFile=', 'jobName=', 'jobDescription=', 'configName=', 'jobGroupName=', 'jobTypeName=', 'username=', 'userAlias=', 'branch=', 'buildInfoName=', 'buildNumber=', 'machinepoolName=', 'machinepoolDescription=', 'architectureName=', 'manufacturerName=' , 'microarchName=', 'numberOfCores=', 'numberOfLogicalProcessors=', 'totalPhysicalMemory=', 'osInfoName=', 'osVersion=', 'machineName=', 'machineDescription='])
+    optdict = dict(opts)
+    check_required_args(optdict)
+    return optdict
+
+def main():
+    """
+        Usage::
```

Comment:
Thanks @jorive we are planning to add the machine specific info as a part of v2, will make a note of this